.pre-commit-config.yaml

```diff
@@ -29,21 +29,21 @@ repos:
     rev: 21.12b0
     hooks:
     -   id: black
-        args: [--check]
+        args: [--check, --diff]
 -   repo: https://github.com/PyCQA/flake8
     rev: 4.0.1
     hooks:
     -   id: flake8
+        args: [--exclude, tests/test_cases/explicit_signature.py]
```

Comment:
Please remove this line, instead special ``# noqa ...`` comments to ``tests/test_cases/explicit_signature.py`` as needed.